nrf_modem/doc/delta_dfu.rst

```diff
@@ -22,7 +22,7 @@ Delta update packages
 
 A delta update is a binary file, with a :file:`.bin` extension.
 The update file is named in such a way that both the source and destination firmware versions can be identified, for example, :file:`mfw_nrf9160_update_from_1.2.2_to_1.2.3`.
-Modem delta update files are part of the official modem releases and can be found in the `nRF9160 modem firmware zip file`_.
+Modem delta update files are part of the official modem releases and can be found in the `nRF91 Series modem firmware zip file`_.
```

Comment:
I don't think we can simply rename the links, there are going to be separate links for modem firmware zip files.